docs/reference/contrib/legacy_richtext.md

```diff
@@ -0,0 +1,61 @@
+# Legacy richtext
+
+Potato `(っ◕‿◕)っ`
+
+Test eval RST
```

Comment:
This and below currently cause a warning in the Sphinx build, but as far as I can see it’s only because of the content being duplicated between two files rather than any syntax issue.

Reply:
One added note – I think the whole premise of switching to Markdown is to make documentation contributions more approachable, so I’d be keen to hear people’s thoughts on whether the RST embeds are desirable in that respect, or whether we’d rather rewrite docs to rely less on this more complex formatting